telos-backend/src/routes/api/journal.js

```diff
@@ -0,0 +1,48 @@
+const express = require('express');
+
+const router = express.Router();
+
+// get widgets which appear in the journal for a date range.
+// query params:
+// 'widget': type of widget i.e. 'all', 'calendar', etc.
+// 'startDate': start date
+// 'endDate': end date
+router.get('/:widget', (req, res) => {
```

Comment:
For the journal, I don't think we need to include the widget type as a param since it will always be retrieving all widgets for that date. Also, Do we need a date range for the journal? or just a date since pages in the journal are always one day.